src/python/dart/util/logging_utils.py

```diff
@@ -0,0 +1,30 @@
+import logging
+from flask import g
+
+class DartLogger(logging.getLoggerClass()):
```

Comment:
It looks like this is going the direction of MDC (mapped diagnostic context) logging, common in the java world.  I would consider something like this: https://gist.github.com/mdaniel/8347533 as a more generic solution.